packages/account/src/Sections/Profile/PersonalDetails/__tests__/personal-details-form.spec.tsx

```diff
@@ -44,6 +44,7 @@ describe('<PersonalDetailsForm />', () => {
                 place_of_birth: 'Thailand',
                 citizen: 'Thailand',
                 email_consent: 1,
+                last_name: 'Doe',
```

Comment:
why are we adding last_name here if we are not checking 🙏

Reply:
Removed it